test-packages/test-utils/src/utils/download-ui5-resources.ts

```diff
@@ -0,0 +1,78 @@
+import { TestModelVersion } from "../../api";
+import { reduce, keys, map } from "lodash";
+import { resolve } from "path";
+import { writeFile, mkdirs, pathExists } from "fs-extra";
+import fetch from "node-fetch";
+
+async function getLibs(version: TestModelVersion): Promise<string[]> {
+  // The metadata.json seems to have been added only very recently :(
+  let versionInURL: string = version;
+  if (versionInURL !== "1.76.0") {
+    versionInURL = "1.76.0";
+  }
+  const url = `https://unpkg.com/@sapui5/distribution-metadata@${versionInURL}/metadata.json`;
+  const response = await fetch(url);
+  if (!response.ok) {
+    console.log(`error fetching from ${url}`);
+    return [];
+  }
+  const fileContent = await response.text();
+  const librariesMetadata = JSON.parse(fileContent);
+  return keys(librariesMetadata.libraries);
+}
+
+export async function addUi5Resources(
+  version: TestModelVersion,
+  folder: string
+): Promise<void> {
+  // CDN libraries (example URL):
+  // https://sapui5-sapui5.dispatcher.us1.hana.ondemand.com/test-resources/sap/m/designtime/api.json
```

Comment:
is this comment still relevant? (the `us1` based url)

Reply:
I prefer to leave it here until we know for sure what to do in case there is no defined library version (right now we use a default but when we take it from the project, I think that if it's not defined, it will use the latest which is the CDN url in runtime... but I'm not sure).

Reply:
I meant why does the CDN link has a specific **region** (US) included?

Reply:
I think it has to have some region... if you know a URL without a region we can change the comment (but then I assume it will be the EU region or something)

Reply:
@petermuessig shouldn't this be the region-free entry URL so that the automatic server selection is active?